ecosystem/sep-0010.md

```diff
@@ -106,7 +108,9 @@ Request Parameters:
 
 Name      | Type          | Description
 ----------|---------------|------------
-`account` | `G...` string | The stellar account that the wallet wishes to authenticate with the **Server** 
+`account` | `G...` string | The stellar account that the **Client** wishes to authenticate with the **Server** 
+`memo` | string | (optional) The memo to attach to the challenge transaction, for `hash` this should be base64-encoded. See the [Memo](#memos) section for details.
```

Comment:
❗ We should use hex encoding for the hash. That's what we do in the lab, and it is more common in general outside of Stellar to hex encode hashes when a string representation is needed.